x-pack/plugins/observability/public/pages/rule_details/translations.ts

```diff
@@ -41,52 +41,6 @@ export const CREATED_WORD = i18n.translate('xpack.observability.ruleDetails.crea
   defaultMessage: 'Created',
 });
 
-export const confirmModalText = (
```

Comment:
These functions were:
* only used in `DeleteConfirmationModal` and nowhere else
* `DeleteConfirmationModal` was set up in a way where it could only delete one thing at a time, so de facto `numIdsToDelete` would always be `1`
* Since it would only allow the deletion of 1 thing at a time, it made no sense to even pass a `multipleTitle`
* By removing that constraint, it became possible to increase the legibility of the `i18n` string by writing  `"You can't recover {title} after deleting."`